x-pack/legacy/plugins/siem/server/lib/detection_engine/rules/create_rules.ts

```diff
@@ -64,15 +67,16 @@ export const createRules = ({
         riskScore,
         severity,
         threat,
+        throttle,
         to,
         type,
         references,
         version,
       },
       schedule: { interval },
       enabled,
-      actions: [], // TODO: Create and add actions here once we have email, etc...
-      throttle: null,
+      actions,
+      throttle: getAlertThrottle(throttle),
```

Comment:
As indicated above, I would remove this function and not overload with our own sentinel values or add new values for branching logic to the throttle and I would remove throttle from the params everywhere.